project/webapp/model/productModel.js

```diff
@@ -15,6 +15,27 @@ sap.ui.define([
 
       getModel() {
         return this.oModel;
-      }
+      },
+
+      removeProductsFromModel(aSelectedProducts) {
```

Comment:
the method can be named just as removeProducts
param name should name as aProductId